cips/cip-25.md

```diff
@@ -17,9 +17,10 @@ This Meta CIP lists the CIPs included in the Ginger network upgrade.
 
 ### Included CIPs
 
+- [CIP-10](./cip-10.md): Coordinated network upgrades
```

Comment:
CIP-10 was already included in Lemongrass so IMO shouldn't be listed here. It's fine to list as a prerequisite at the top (next to requires) if you think it's needed



```suggestion
```

Reply:
added this but accidentally didn't use the suggestion, my b!